abixen-platform-business-intelligence-service/src/main/java/com/abixen/platform/service/businessintelligence/multivisualisation/domain/model/impl/datasource/file/FileDataSourceRowBuilder.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2010-present Abixen Systems. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.abixen.platform.service.businessintelligence.multivisualisation.domain.model.impl.datasource.file;
+
+import com.abixen.platform.common.domain.model.EntityBuilder;
+import com.abixen.platform.service.businessintelligence.multivisualisation.domain.model.impl.datasource.DataSource;
+
+public class FileDataSourceRowBuilder extends EntityBuilder<FileDataSourceRow> {
+
+    public FileDataSourceRowBuilder rowNumeber(final Integer rowNumber) {
```

Comment:
should be added after assemble method

Reply:
Do you use this builder only for test? If yes, remove. We don't create code for tests. We create tests for core.